internal/impl/dashboard.go

```diff
@@ -0,0 +1,528 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package impl
+
+import "fmt"
+
+// grafanaDashboard generates the content of a Service Weaver dashboard that
+// contains autogenerated metrics and the logs for a given app.
+//
+// TODO(rgrandl): see how we can display the traces as well. Traces is more
+// complicated because it requires to specify a service name, which is not
+// available before the application is running.
+func grafanaDashboard(app string) string {
+	return fmt.Sprintf(`{
```

Comment:
Can you add a comment about how this was generated?

Also, can you store this text in a separate txt file, and then get it via `//go:embed`?

Reply:
Done